tests/test_moving_eddies.py

```diff
@@ -61,8 +61,8 @@ def cosd(x):
         U[:, :-1, t] = np.diff(P[:, :, t], axis=1) / dy / corio_0 * g
         U[:, -1, t] = U[:, -2, t]  # Fill in the last row
 
-    return Grid.from_data(U, lon, lat, V, lon, lat,
-                          depth, time, field_data={'P': P})
+    return Grid.from_data(U, lon, lat, V, lon, lat, depth, time, field_data={'P': P},
+                          u_units=GeographicPolar(), v_units=Geographic())
```

Comment:
I think we should have simply one `units=GeographicPolar()` here, because we can assume that U and V have the same grid. Also, I think that `units=GeographicPolar()` should be the default (i.e. if you don't provide units, it should assume GeographicPolar()